lib/hunter/commands/hunt.rb

```diff
@@ -173,22 +174,30 @@ def process_packet(data:)
 
         EOF
 
+        buffer = NodeList.load(Config.node_buffer)
+        parsed = NodeList.load(Config.node_list)
```

Comment:
The lists are also being loaded in the first two lines of `#process_packet`. Is this intentional?

Reply:
Not intentional, just a remnant from copy/pasting.